src/StyledComponents.js

```diff
@@ -0,0 +1,75 @@
+import { styled } from "@mui/material/styles";
+import Paper from "@mui/material/Paper";
+import customTheme from "./theme";
+import { Box, Button, Divider, Link } from "@mui/material";
+
+const ContactItem = styled(Paper)(({ theme }) => ({
+  padding: theme.spacing(1),
+  textAlign: "center",
+}));
+export default ContactItem;
+
+const HeroTitle = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(1),
+  variant: "h3",
```

Comment:
Keep all props separate from style, like "variant" 
I think Box don't have "variant" as props, can you keep it Box, but for the text inside it, use the Variant component

Reply:
done